admin-site/src/App.svelte

```diff
@@ -0,0 +1,421 @@
+		document.getElementById("openEvent").style.display = "block";
+	}
+
+	function confirmOccasionDelete(){
+		document.getElementById("closeEvent").style.display = "none";
+		document.getElementById("confirmDelete").style.display = "block";
+	}
+	function cancelDelete(){
+		document.getElementById("confirmDelete").style.display = "none";
+		document.getElementById("closeEvent").style.display = "block";
+	}
+
+	function deleteOccasion(){
+		// deletion code goes here
+		document.getElementById("confirmDelete").style.display = "none";
+		document.getElementById("eventsList").style.display = "block";
+		//removes occasion from object and from list
+		focusedEvent.occasions.splice(indexInOccasions,1);
+		document.getElementById(focusedOccasionID).remove();
+
+	}
+	function backToEvents(){
+		let id =this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("eventsList").style.display = "block";
+	}
+	function cancelEnd(){
+		
+		document.getElementById("confirmEndOccasion").style.display = "none";
+		document.getElementById("openEvent").style.display = "block";
+	}
+	function confirmEnd(){
+		
+		document.getElementById("openEvent").style.display = "none";
+		document.getElementById("confirmEndOccasion").style.display = "block";
+	}
+
+	function backToOccasionList() {
+		let id = this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("occasionList").style.display = "block";
+		
+	}
+	function showQR(){
+		let id = this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("QRcode").style.display = "block";
+
+	}
+
+//this changes which cue details are shown
+	function changeCueState(){
+		let direction = this.value;
+		if(direction == "next" && cueState <= focusedEvent.cues.length-1){
+			cueState += 1;
+		} else if(direction == "previous" && cueState > 1){
+			cueState -= 1;
+		} else {
+			cueState = 1;
+		}	
+	}
+
+/////for new event generation
+	function setTitle(event){
+		label = event.target.value;
+	}
+
+//whichever parameters we want to be able to build for new events from the site would go below
+	function createEvent(){
+		let newEvent = [{
+				label:label
+			}];
+		events = events.concat(newEvent);
+	}
+////////
+</script>
+
+<style>
+	#eventsList, #closeEvent,#openEvent, #occasionList, #QRcode, #confirmDelete, #confirmEndOccasion {
+		display: none;
+	}
+	
+	
+</style>
+
+
+<!-- Keeping this as a component cause it will likely need switching out -->
+<section id = "login">
+	<Login />
+
+</section>
+
+<!-- #eventsList allows a list of events to be built and shown -->
+
+<section id = "eventsList">
+	<h1 class="text-center">Events</h1>
+	<hr>
+		{#if events.length === 0}
+			<p>No events have been added yet</p>
+			{:else}
+					{#each events as event}
+						<button type="button" class= 'btn btn-primary btn-block' value = {event.id} on:click={eventButton} >
+							<h3>{event.label}</h3>	
```

Comment:
With bootstrap, when you use styled text inside a button you often have to kill margins to get the baseline to look right. <img width="430" alt="Screen Shot 2019-10-31 at 12 29 48 PM" src="https://user-images.githubusercontent.com/4842862/67966419-3d377e00-fbda-11e9-946c-7f5e636a7f06.png"> 

This text is too high within the button.

You can try something like
```
.button > h3 {
  margin-bottom: 0;
}
```

Reply:
change applied